astropy/cosmology/_src/flrw/base.py

```diff
@@ -35,19 +35,23 @@
     validate_non_negative,
     validate_with_unit,
 )
+from astropy.cosmology._src.traits import (
+    ScaleFactor,
+    TemperatureCMB,
+    _BaryonComponent,
+    _CriticalDensity,
+)
 from astropy.cosmology._src.utils import (
     aszarr,
     deprecated_keywords,
     vectorize_redshift_method,
 )
 
 if TYPE_CHECKING:
-    from collections.abc import Callable, Mapping
+    from collections.abc import Mapping
     from typing import Self
 
-    from numpy.typing import ArrayLike, NDArray
-
-    from astropy.units import Quantity
+    import astropy.units
```

Comment:
Why is this being changed?

Reply:
Sphinx shenanigans in resolving imports!
https://tenor.com/vdpH.gif